docker/test/integration/cluster/checkers/AwsChecker.py

```diff
@@ -29,6 +29,18 @@ def check_s3_server_object_data(self, container_name, test_data):
         (code, file_data) = self.container_communicator.execute_command(container_name, ["cat", s3_mock_dir + "/binaryData"])
         return code == 0 and file_data == test_data
 
+    @retry_check()
+    def check_s3_server_object_hash(self, container_name: str, expected_file_hash: str):
+        (code, output) = self.container_communicator.execute_command(container_name, ["find", "/s3mockroot/test_bucket", "-mindepth", "1", "-maxdepth", "1", "-type", "d"])
+        if code != 0:
+            return False
+        s3_mock_dir = output.strip()
+        (code, md5_output) = self.container_communicator.execute_command(container_name, ["md5sum", s3_mock_dir + "/binaryData"])
+        if code != 0:
+            return False
+        file_hash = md5_output.split(' ')[0].strip()
+        return code == 0 and file_hash == expected_file_hash
```

Comment:
we could remove the second `code == 0` check now:
```suggestion
        if code != 0:
            return False
        file_hash = md5_output.split(' ')[0].strip()
        return file_hash == expected_file_hash
```